services/minespace-web/src/components/pages/Project/InformationRequirementsTablePage.tsx

```diff
@@ -138,6 +134,7 @@ export const InformationRequirementsTablePage = () => {
   }, [uploadedSuccessfully]);
 
   useEffect(() => {
+    handleFetchData();
     return () => {
       dispatch(clearInformationRequirementsTable({}));
```

Comment:
fun fact- clearing the IRT means that if you click the back button from the IRT to return to the project overview, your IRT now (on test) shows "Not Started". 
So, a couple changes here-
- combine useEffects with same dep arrays
- it doesn't need to fetch the data on load, because it's a child of the project page
- project page looks at the project.irt instead of the irt in the state because I didn't like all the refreshing I had to do.
- but I didn't totally refactor the data stuff here, it was just too much